cspell.schema.json

```diff
@@ -92,12 +149,12 @@
       "additionalProperties": false,
       "properties": {
         "allowCompoundWords": {
-          "default": false,
+          "defaultValue": false,
```

Comment:
Not sure if this is going to cause issues

Reply:
This file is auto generated. "default" is part of the json-schema spec.